code/game/gamemodes/dynamic/dynamic.dm

```diff
@@ -615,6 +615,10 @@ GLOBAL_VAR_INIT(dynamic_forced_threat_level, -1)
 		var/midround_injection_cooldown_middle = 0.5*(midround_delay_max + midround_delay_min)
 		midround_injection_cooldown = (round(clamp(EXP_DISTRIBUTION(midround_injection_cooldown_middle), midround_delay_min, midround_delay_max)) + world.time)
 
+		// Time to inject some threat into the round
+		if(SSevacuation.station_evacuated()) // Unless the shuttle is past the point of no return
```

Comment:
This used to be a point of no return check, not a post-evac check.

Reply:
Right now `station_evacuated()` is a point of no return check. Though if you want, I can change `station_evacuated()` to check for shuttle arriving to the CentCom instead of PONR